examples/asr/rate_punctuation.py

```diff
@@ -0,0 +1,207 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+from argparse import ArgumentParser
+
+import pandas as pd
+from tabulate import tabulate
+
+from nemo.collections.common.metrics.per import PERData
```

Comment:
Why is this at root of the example directory? It is not a universal script, applicable only to PC models no ?

Better to put under some directory such as "scoring" (better name please)

Reply:
If it's universal, merge the logic with the current eval speech script. And guard all the extra dependencies like tabulate etc in that case

Reply:
> Better to put under some directory such as "scoring" (better name please)

@titu1994, does it make sense to move it to `https://github.com/NVIDIA/NeMo/blob/main/scripts/metric_calculation/`?

Reply:
need to remove file

Reply:
@karpnv @titu1994 thank you! the script rate_punctuation has been removed, and per calculation has been added to speech_to_text_eval.py. Could I ask for a review?